src/app.rs

```diff
@@ -41,6 +41,9 @@ pub struct Config<'a> {
     /// The character width of the terminal
     pub term_width: usize,
 
+    /// The width of tab characters.
+    pub tab_width: usize,
```

Comment:
How is "no tab expansion" represented? `tab_width==0`? Could this be `Option<usize>` instead?

Reply:
It probably could be, but it would probably complicate the argument parsing a little bit.

Reply:
I would favor the `Option` version but I'm also fine with a short comment for now ("A value of zero indicates that no expansion should be performed").

Reply:
Added a comment in d404139